DTS-STN/Alpha-Site
# Update signup page content

# Description

[Content update - signup form edition](https://trello.com/c/SWSpQsId)

This PR updates the layout and content of the signup form. Unfortunately it required a new component so that's been bundled in here as well. I'm sorry this is so huge.

## Acceptance Criteria

The signup page matches what is in the designs.

## Test Instructions

1. Navigate to the signup form
2. Verify that the content/layout matches the designs in Figma

## Help Requested

- **There is something not working quite right with spacing in one checkbox lists - _please advise_**

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
